doc/manuals/orion-api.md

```diff
@@ -585,6 +693,425 @@ NGSIv2 `error` reporting is as follows:
   + HTTP 413 Request Entity Too Large corresponds to `RequestEntityTooLarge` (`413`)
   + HTTP 415 Unsupported Media Type corresponds to `UnsupportedMediaType` (`415`)
 
+## Update operators for attribute values
+
+Some attribute value updates has special semantics, beyond the ones described in the
+NGSIv2 specification. In particular we can do requests like this one:
+
+```
+POST /v2/entities/E/attrs/A
+{
+  "value": { "$inc": 3 },
+  "type": "Number"
+}
+```
+
+which means *"increase the value of attribute A by 3"*.
+
+This functionality is useful to reduce the complexity of applications and avoid
+race conditions in applications that access simultaneously to the same piece of
+context. More detail in [specific documentation](user/update_operators.md).
+
+### Supported operators
+
+Orion update operators are based on a subset of the ones implemented by MongoDB
+(described [here](https://docs.mongodb.com/manual/reference/operator/update/)). The 
+complete set of operators supported by Orion are the following:
+
+| Operator                 | Previous attr value     | Operation                          | Final value                     |
```

Comment:
It's a bit weird to provide examples of the operation before having described the operator, but I think is fine.

Reply:
Ok, NTC